pom.xml

```
@@ -39,8 +39,8 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <okhttp3.version>3.14.9</okhttp3.version>
-        <okhttp4.version>4.10.0</okhttp4.version>
+        <okhttp3.version>4.11.0</okhttp3.version>
```

This is not relevant
```suggestion
        <okhttp3.version>3.14.9</okhttp3.version>
```